tests/functional_tests/0006_set_test.py

```diff
@@ -22,21 +22,73 @@
 # IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 import pytest
-from click.testing import CliRunner
 
-import iocage_cli as ioc
 
 require_root = pytest.mark.require_root
 require_zpool = pytest.mark.require_zpool
 
 
+# Let's perform testing first on different types of jails
+# Then we can move on to setting some specific properties and seeing
+# do the props work as intended
+
+
+# TODO: Plugin test left
+
+
+def _set_and_test_note_prop(invoke_cli, value, config_data, jail):
+    invoke_cli(
+        ['set', f'notes={value}', jail]
+    )
+
+    assert (config_data(jail) or {}).get('notes') == value,\
+        f'Failed to set note value to {value}'
+
+
+@require_root
+@require_zpool
+def test_01_set_prop_on_thin_jail(thin_jails, invoke_cli, config_data):
```

Comment:
Calling these thin jails are confusing as historically that had meaning, is confusing with the thin/thick configuration types and also because it's a moot point past creation. iocage doesn't treat them any differently under the hood.
```suggestion
def test_01_set_prop_on_jail(jails, invoke_cli, config_data):
```